### ISSUE TITLE
Test result not looking good on fashion dataset (blurry faces), any ideas?

### ISSUE THREAD

**Issue by Lotayou** [AliaksandrSiarohin/pose-gan#13]
Hi I just run the test on both fashion dataset with all of your pre-trained models. For all cases I used the same parameter as in `full` mode, like for fashion dataset:

`
export CUDA_VISIBLE_DEVICES=1 
python test.py --generator_checkpoint path/to/generator/checkpoint --output_dir output/full --checkpoints_dir output/full --warp_skip mask --dataset fasion --l1_penalty_weight 0.01 --nn_loss_area_size 5 --batch_size 2 --content_loss_layer block1_conv2 --number_of_epochs 90
`

The result looks okay for market images (probably due to the fact that it's low-resolution and blurry in the first place). But for fashion dataset I can notice severe degradation of human faces: distorted contour, blurry eyes/nose/mouth, etc. It's not as good as the result posted in your paper. Here I show some results of generator-warp-mask-nn5-cl12.h5, and using any other checkpoints will yield similar result.

![fasionmenjacketsvestsid0000243904_1front jpg_fasionmenjacketsvestsid0000243904_2side jpg](https://user-images.githubusercontent.com/33449901/43627477-2810f6c6-9729-11e8-90ca-9215968c4f83.png)
![fasionwomencardigansid0000368603_2side jpg_fasionwomencardigansid0000368603_3back jpg](https://user-images.githubusercontent.com/33449901/43627503-3e6ae224-9729-11e8-9126-0526fabbfa79.png)
![fasionwomenteestanksid0000665502_2side jpg_fasionwomenteestanksid0000665502_1front jpg](https://user-images.githubusercontent.com/33449901/43627631-b628f684-9729-11e8-9b51-a838c7a0686a.png)

I'm afraid that I used the wrong parameter for testing. Can you tell me what's the correct testing setting? Also, which checkpoint gives the best result in your experiment? I can't tell that from the generated result, since they all have the same problem of blurry faces.

**Comment by AliaksandrSiarohin** [comment]
In the paper I select the images that look the best.
Images of men is bad because there is a dataset bias (number of women images is much larger that number of men).
The faces looks worse than the rest, because human tend to concentrate their attention on the faces.

**Comment by Lotayou** [comment]
I see. So there’s nothing wrong with the way I ran the test, right? Well I guess that generating high resolution image is indeed a challenging task then.
